src/components/MainSection/index.tsx

```diff
@@ -32,6 +45,12 @@ function MainSection({
     changeContinent(continent);
     setShowFilter(false);
   }
+  const { data, isLoading } = useQuery({
+    queryKey: ["countries"],
+    queryFn: fetchAllCountries,
+  });
+  console.log("Here is the data");
+  console.log(data);
```

Comment:
@jakemelvin is it necessary to leave the `console.log`?

Reply:
I agree. Never push logs